src/lib/tasks/run-texture-sync.ts

```diff
@@ -217,4 +219,7 @@ export function runTextureSync(coordinator: CoordinatorAPI): void {
     height: rowHeight,
   };
   coordinator.targetValuesTexture.subimage(subimageData, 0, lowRow);
+
+  // After texture sync, we should always guarantee a draw call.
+  coordinator.queueDraw();
```

Comment:
This is the more correct solution to ensuring that sprite's are always rendered in their updated state. Really, the Texture sync task should have always been queueing a draw.

In most cases, a draw would have already been queued. This line covers the rare case where a sprite's transition time is sufficiently short that it should have already arrived where it's going by the time the Texture sync is able to flash its values to the data texture.